packages/kotti-ui/source/kotti-field-number/KtFieldNumber.vue

```diff
@@ -311,6 +311,7 @@ export default defineComponent({
 	display: flex;
 	align-items: center;
 	font-variant-numeric: tabular-nums;
+	margin: 0 -0.5rem;
```

Comment:
you're probably counter-acting this line https://github.com/3YOURMIND/kotti/blob/a90f3150378ea25379f5b825a9642f49c0d2951b/packages/next/kotti-field/KtField.vue#L176

if so, either link this in a comment so we always have the line, or even better, make some variable for the input-container padding so that when we change this in the future it's immediately updated for the `KtFieldNumber`

obviously 0.5 is not 0.8 but i guess u get my point still, you make it 5/8th of it, I don't mind, whatever u see fit
> my point is merely that it requires one LOC change in the future if need be :)